Makefile

```
@@ -0,0 +1,2 @@
+tests:
```

Should we use the (afaict) more standard `test` target?

```suggestion
test:
```